pam/integration-tests/gdm_test.go

```diff
@@ -296,6 +296,31 @@ func TestGdmModule(t *testing.T) {
 				{Access: auth.Granted},
 			},
 		},
+		"Authenticates_user_switching_to_phone_ack_from_auth_mode_selection_stage": {
```

Comment:
Typo in commit message:

"where an use may" > "where a user may"

Reply:
Thanks, fixed!